pages/about.vue

```diff
@@ -145,6 +145,21 @@
           </a>
         </li>
       </ul>
+      <p>
+        {{
+          $t(
+            'Google Analyticsによる情報送信を回避する場合は、Google がサポートする'
+          )
+        }}
+        <a
+          :href="$t('https://tools.google.com/dlpage/gaoptout?hl=ja')"
+          target="_blank"
+          rel="noopener"
+        >
+          {{ $t('測定を無効にするブラウザ アドオン') }}
+        </a>
+        {{ $t('をご利用ください。') }}
+      </p>
```

Comment:
文中にリンクを挿入する場合は、以下のように `<i18n>` コンポーネントを使用すると、語順が日本語と異なる言語にも対応できます。
また、 #1961 にて `noreferrer` を追加しているので、ここも同様にするべきかなと思います。
```suggestion
      <i18n
        tag="p"
        path="Google Analyticsによる情報送信を回避する場合は、Google がサポートする{addon}をご利用ください。"
      >
        <template v-slot:addon>
          <a
            :href="$t('https://tools.google.com/dlpage/gaoptout?hl=ja')"
            target="_blank"
            rel="noopener noreferrer"
          >
            {{ $t('測定を無効にするブラウザ アドオン') }}
          </a>
        </template>
      </i18n>
```
（ `ja.json` の翻訳キーも同じように変更する必要あり）

Reply:
ご教示ありがとうございます！
対応しました